iot_api/iotservice.py

```diff
@@ -14,7 +14,7 @@
 from iot_api import app, api, jwt
 from iot_api.user_api import db
 from iot_api import mail, socketio
-from iot_api.user_api.model import RevokedTokenModel
+from iot_api.user_api.models.RevokedTokenModel import RevokedTokenModel
```

Comment:
Cannot be imported directly from iot_api.user_api.models?

`from iot_api.user_api.models import RevokedTokenModel`

Reply:
You are right, done!